apps/authentication/routes.py

```diff
@@ -139,6 +138,32 @@ def register():
     else:
         return render_template('pages/register.html', form=create_account_form)
 
+@blueprint.route('/groups/create', methods=['GET', 'POST'])
+def create_group():
+    form = GroupForm()
+    students = Users.query.filter_by(category='student').all()
+    
+    if form.validate_on_submit():
```

Comment:
if the form is not valid, we should provide means to send a message for the user saying why the form was considered invalid